simpl/db/mongodb.py

```diff
@@ -115,6 +115,55 @@ def scrub(data):
         raise ValidationError("Input '%s' not permitted: %s" % (data, exc))
 
 
+def build_text_search(strings, name_field='name'):
```

Comment:
And what about text score? This seems useful. Maybe `sort=True` could return the results sorted highest-to-lowest by textscore?

http://docs.mongodb.org/master/reference/operator/query/text/#text-score

We could revisit this later too, but I assume these will be requests from the UI and should probably be sorted by a score like this.